mappings/net/minecraft/server/config/OldListsConverter.mapping

```diff
@@ -0,0 +1,42 @@
+CLASS none/mq net/minecraft/server/config/OldListsConverter
```

Comment:
I think on a similar class for migration between versions "Upgrader" was chosen, right @copygirl?